src/utils/type/isVoid.js

```diff
@@ -0,0 +1,7 @@
+// @flow
+
+function isVoid(data: any): boolean {
+  return (data == null)
```

Comment:
Why are you using round brackets?
Using the abstract equality (`==`) operator is deliberate?

Reply:
(data == null) checks null or undefined